### ISSUE TITLE
Can I training your code with V100 GPU

### ISSUE THREAD

**Issue by lijiannuist** [TRI-ML/prismatic-vlms#21]

**Comment by siddk** [comment]
You should be able to; based on the memory available on the V100 cards though, you may need to cut the batch size, or turn on gradient accumulation.
